etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/results/SendResultUseCase.java

```diff
@@ -41,15 +41,15 @@ public void convertAndSend(Result<?> result, String receivedSubmissionId)
                         result.getReceivingFacilityDetails(),
                         result.getPlacerOrderNumber());
 
-        sendMessageHelper.savePartnerMetadataForReceivedMessage(partnerMetadata);
+        sendMessageHelper.savePartnerMetadataForInboundMessage(partnerMetadata);
 
         transformationEngine.runRules(result);
 
         String outboundReportId = sender.send(result).orElse(null);
-        logger.logInfo("Sent result reportId: {}", outboundReportId);
+        logger.logInfo("Outbound result outboundReportId: {}", outboundReportId);
```

Comment:
I think we can keep "Sent result outboundReportId" here because it happens right after sending the result to RS

Reply:
That makes sense